packages/components/src/menu-group/index.js

```diff
@@ -33,11 +28,17 @@ export function MenuGroup( {
 	return (
 		<div className={ classNames }>
 			{ label &&
-				<div className="components-menu-group__label" id={ labelId }>{ label }</div>
+				<div
+					className="components-menu-group__label"
+					id={ labelId }
+					aria-hidden="true"
```

Comment:
I'm not really sure at what point this attribute was added. Do you recall the rationale? In any case, from testing in screen reader, it doesn't seem to affect whether the inputs in the group are announced as being associated with the group (Chrome + macOS Voiceover).

![image](https://user-images.githubusercontent.com/1779930/58581374-66b04000-821d-11e9-9c1c-c04e683897fa.png)

Reply:
I guess this was added to avoid a double announcement. The group _should_  be announced with its name because `aria-labelledby` targets the visible label. Actually, checking this kind of stuff in the Chrome dev tools is now pretty easy. Go in Elements > right panel > accessibility tab, select the element and check its accessible name. It also tells you where the accessible name comes from.

Should be tested with other screen readers on Windows but looks good to me at a first glance.